app/api/api/routers.py

```diff
@@ -0,0 +1,122 @@
+from api import views
+from rest_framework import routers
+
+routers = routers.DefaultRouter()
```

Comment:
⛏️ can we rename `routers` to `router` seeing for the class is `DefaultRouter`?

Reply:
Yep good point - have pushed this change